### PR TITLE
add library registration guide for .NET6

### PR DESCRIPTION
**Pull request by prusikt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
...
+### Unregister-> .NET 6.0:
+- Run cmd.exe as administrator, change directory location to  dscom.exe location
+- Change directory to the location of regasm.exe (Registration Assembly Tool)
+- UnRegister MAT.Atlas.Automation.Api.dll and MAT.Atlas.Automation.Client.dll with the commands below.
+
+ 1. 'dscom.exe tlbunregister "{{full file path to MAT.Atlas.Automation.Api.dll}}"'
+ 2. 'dscom.exe tlbunregister "{{full file path to MAT.Atlas.Automation.Client.dll}}"'
+
+### Debug and Run
+a. Launch ATLAS 10
```
Comment (owentmfoo) [inline README.md]: list still need fixing

File: README.md [hunk 2]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
...
+vary depending upon the exact .Net version installed.
+- Register MAT.Atlas.Automation.Api.dll with the command below.
+- Register MAT.Atlas.Automation.Client.dll with the command below.
+
+### Unregister-> .NET 6.0:
+- Run cmd.exe as administrator, change directory location to  dscom.exe location
+- Change directory to the location of regasm.exe (Registration Assembly Tool)
+- UnRegister MAT.Atlas.Automation.Api.dll and MAT.Atlas.Automation.Client.dll with the commands below.
+
+ 1. 'dscom.exe tlbunregister "{{full file path to MAT.Atlas.Automation.Api.dll}}"'
```
Comment (owentmfoo) [inline README.md]: CLI commands should be marked by `` ` `` instead of `'`

File: README.md [hunk 3]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
 - For SQLite: DBEngine=SQLite;Data Source=path\to\file.ssn2" 
 (<sup>2</sup>) You will need to reload the session after each run, as the same transient parameter cannot be added twice. 
 
-ATLAS Automation API is available as a Nuget package to registered users from the **[McLaren Applied Nuget Repository](https://github.com/mat-docs/packages)**.
\ No newline at end of file
+ATLAS Automation API is available as a Nuget package to registered users from the **[McLaren Applied Nuget Repository](https://github.com/mat-docs/packages)**.
+
+
+<img src="/images/malogo.png" width="300" align="right" /><br><br><br>
```
Comment (owentmfoo) [inline README.md]: logo already present at L1

File: README.md [hunk 4]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
...
+
+ 1. 'dscom.exe tlbexport "{{full file path to MAT.Atlas.Automation.Client.dll}}"'
+ 2. 'dscom.exe tlbregister "{{full file path to newly created tbl file}}"'
+
+### Unregister-> .NET Framework
+- Run cmd.exe as administrator
+- Change directory to the location of regasm.exe (Registration Assembly Tool)
+N.B. this can be found in the .Net Framework 4 installation folder. This version number may
+vary depending upon the exact .Net version installed.
+- Register MAT.Atlas.Automation.Api.dll with the command below.
```
Comment (owentmfoo) [inline README.md]: command missing?

File: README.md [hunk 5]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
...
\ No newline at end of file
+ATLAS Automation API is available as a Nuget package to registered users from the **[McLaren Applied Nuget Repository](https://github.com/mat-docs/packages)**.
+
+
+<img src="/images/malogo.png" width="300" align="right" /><br><br><br>
+
+# ATLAS 10 COM API Migration GUIDE!
+
+* **9.2.3 APIs and Libraries**
+This example references the following libraries:
```
Comment (owentmfoo) [inline README.md:28]: do you intended this to be starting on a new line?

File: README.md [hunk 6]
```diff
@@ -18,4 +18,74 @@ Both sample codes do the same<sup>1</sup>: creating a transient parameter, addin
...
+- MAT_Atlas_Automation_Client
+- MAT_Atlas_Automation_Api
+N.B. before these can be used they first must be registered. It is planned for this to be part of the installer once officially released. See Registering Automation API DLLs.
```
Comment (owentmfoo) [inline README.md:32]: do you intended this to be starting on a new line?
These dlls are already part of the installer

File: README.md (hunk 5 above)
Comment (prusikt) [reply README.md:28]: yes, I did

File: README.md (hunk 6 above)
Comment (prusikt) [reply README.md:32]: yes I did

**Review by owentmfoo — APPROVED** [review]

**Pull request merged**